replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/AbstractPersistable.java

```diff
@@ -55,6 +60,16 @@ public void setVersion(int version) {
     this.version = version;
   }
 
+  @Override
+  public Date getModified() {
+    return modified;
+  }
+
+  @Override
+  public void setModified() {
```

Comment:
❓ Should we expose this one? I am thinking it should be removed or if left there for testing, then let's make it package private and annotate accordingly.